bundle/Command/GenerateProjectCommand.php

```diff
@@ -153,11 +154,48 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
         // Various cleanups
         $this->cleanup();
 
+        $this->activateGitHooks();
+
         $this->writeSection(['You can now start using the site!']);
 
         return 0;
     }
 
+    private function activateGitHooks()
+    {
+        if (!class_exists(Config::class)) {
+            return;
+        }
+
+        /** @var \Symfony\Component\Filesystem\Filesystem $fileSystem */
+        $fileSystem = $this->getContainer()->get('filesystem');
+        $projectDir = $this->getContainer()->getParameter('kernel.project_dir');
+        $captainHookFiles = [
+            $projectDir . '/captainhook_disabled.json',
+            $projectDir . '/captainhook_enabled.json',
+        ];
+
+        if ($fileSystem->exists($captainHookFiles)) {
```

Comment:
Don't you mean `if (!$fileSystem->exists($captainHookFiles)) {` ?

Reply:
Fixed in 4a4e9de60112f01518762d97089a91cd067d175d